### DISCUSSION TITLE
Can't open /don't see a new Pad

### DISCUSSION THREAD

**Discussion by Pfandgiraffe** [otetard/ownpad#148] [Q&A]
Hi, have tried different hosts but after i have created a new Pad inside the file manager it won't open. Afterwards i have reload the hole page and i click on the new Pad a download of the pad starts but nothing more.
My NC is hosted by Hetzner Service. Maybe this is the fault? Thanks for feedback!

**Comment by otetard** [comment]
Hi!

Did you try to manually configure your Nextcloud instance to support custom MIME types used by Ownpad? Basically, you should follow the steps mentioned in the main README file: https://github.com/otetard/ownpad?tab=readme-ov-file#manual-configuration

1. Edit `mimetypealiases.json` and `mimetypemapping.json` files
2. Call `php occ maintenance:mimetype:update-js`
3. Update MIME types for existing files: `sudo -u www-data php occ files:scan --all`
